### ISSUE TITLE
Full UI: Oppia SplashFragment/Activity

### ISSUE THREAD

**Issue by BenHenning** [oppia/oppia-android#11]
The Oppia splash screen should be shown for the duration required for the profile to be loaded and the app fully initialized (database upgrading can block this screen). This is nice-to-have for the proof-of-concept to help introduce a proper app loading flow.

**Comment by BenHenning** [comment]
This is blocked on #4.

**Comment by BenHenning** [comment]
https://xd.adobe.com/view/bee1fced-33bd-4f69-6a89-2c30fe1c4f65-c8d0/

**Comment by BenHenning** [comment]
Design document: https://docs.google.com/document/d/1m9Y5bIdEE8iDM9gElGRYW6PxNuESfHU3EODuZ0z4cJY/edit?usp=sharing.
